src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/XmlSerializerOperationFormatter.cs

```diff
@@ -236,7 +244,8 @@ protected override void GetHeadersFromMessage(Message message, MessageDescriptio
                 if (!bufferReader.IsEmptyElement)
                 {
                     bufferReader.ReadStartElement();
-                    object[] headerValues = (object[])serializer.Deserialize(bufferReader);
+                    //object[] headerValues = (object[])serializer.Deserialize(bufferReader);
```

Comment:
This can be removed entirely.

Reply:
You mean only the commented out line, right? OR all of the `if (!bufferReader.IsEmptyElement)` block?

Reply:
Just the commented out line. No need to keep the incorrect line there in a comment. We have commit history if we want to see what it used to be.